commit-table

```diff
@@ -1,7 +1,9 @@
+968e901aff38a343b1de4addebf79fd8fa991c59 v11.6.0
```

Comment:
Does this change what is "latest"?

Reply:
@shaunmulligan no, this is where we build the node binaries. The latest version is stored in the contract for node stack https://github.com/balena-io/contracts/blob/master/contracts/sw.stack/node/contract.json#L7. Of course, we will change the latest version to v11.6.0 when all binaries are ready